app/api/faucet/route.ts

```diff
@@ -25,37 +25,39 @@ const abi = parseAbi([
 ])
 
 const viemConfig = createConfig({
-  chains: [chain],
+  chains: [arbitrumSepolia],
```

Comment:
not in scope for this PR but we'll probably have a Base sepolia faucet as well, right?